File: quizletservice/src/main/java/com/example/alexeyglushkov/quizletservice/QuizletTerm.java
package com.example.alexeyglushkov.quizletservice;

/**
 * Created by alexeyglushkov on 02.04.16.
 */
public class QuizletTerm {
    private long id;
    private String term;
    private String definition;
    private int rank;

    public long getId() {
        return id;
    }

    public void setId(long id) {
        this.id = id;
    }

    public String getTerm() {
        return term;
    }

    public void setTerm(String term) {
        this.term = term;
    }

    public String getDefinition() {
        return definition;
    }

    public void setDefinition(String definition) {
        this.definition = definition;
    }

    public int getRank() {
        return rank;
    }

    public void setRank(int rank) {
        this.rank = rank;
    }
}

File: quizlet_repository/src/main/java/com/example/alexeyglushkov/quizletservice/entities/QuizletTerm.kt
package com.example.alexeyglushkov.quizletservice.entities

import android.os.Bundle
import android.os.Parcel
import android.os.Parcelable
import android.os.Parcelable.Creator
import kotlinx.android.parcel.Parcelize
import java.io.Serializable

/**
 * Created by alexeyglushkov on 02.04.16.
 */
@Parcelize
data class QuizletTerm(
    var id: Long = 0,
    var setId: Long = 0,
    var term: String? = null,
    var definition: String? = null,
    var rank: Int = 0): Parcelable, Serializable {

    companion object {
        private const val serialVersionUID = -6211744973878309135L
    }
}